src/components/learner-credit-management/BudgetDetailApprovedRequestTable.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DataTable, CheckboxFilter } from '@openedx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import TableTextFilter from './TableTextFilter';
+import CustomDataTableEmptyState from './CustomDataTableEmptyState';
+import RequestDetailsTableCell from './RequestDetailsTableCell';
+import RequestStatusTableCell from './RequestStatusTableCell';
+import RequestAmountTableCell from './RequestAmountTableCell';
+import RequestRecentActionTableCell from './RequestRecentActionTableCell';
+import ApprovedRequestsTableRefreshAction from './ApprovedRequestsTableRefreshAction';
+import { DEFAULT_PAGE, PAGE_SIZE } from './data';
+
+const FilterStatus = (rest) => (
+  <DataTable.FilterStatus showFilteredFields={false} {...rest} />
+);
+
+const getRequestStatusDisplayName = (status) => {
+  if (status === 'waiting_for_learner') {
```

Comment:
From backend we are passing `Waiting for learner` and not `waiting_for_learner`, is it being transformed when the data is fetched?